cds_books/literature/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
```

Comment:
minor: can we name the package plural? I saw now that it is not in ILS, but I/we will change it since we named all records' modules plural.

Reply:
I refactored `cds-books`, I checked also just in case to fix ILS, but its bigger than I thought and few things might break.